app/services/DR/getAllCases.js

```diff
@@ -1,12 +1,30 @@
-import { COV_CASES_SERVICE } from '../../constants/DR/baseUrls';
-import fetch from '../../helpers/Fetch';
-
-export function getAllCases(date) {
-  return fetch(`${COV_CASES_SERVICE}${date ? `?date=${date}` : '/'}`)
-    .then(({ data }) => {
-      return data;
-    })
-    .catch(err => {
-      return err.response.status;
+import Axios from 'axios';
+
+// import { COV_CASES_SERVICE } from '../../constants/DR/baseUrls';
+import { getTokenMepid } from './getToken';
+// import fetch from '../../helpers/Fetch';
+
+export async function getAllCases(date) {
+  let savedToken = await getTokenMepid();
+  const mepidUrl = 'https://covidrdapi.msp.gob.do:8443';
```

Comment:
This credentials should go in the **.env** file and call it from baseUrls. Please delete it 

`Note:` do not commit the .env file